tools/build-prod-env.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+const argv = require('minimist')(process.argv.slice(2))
+const _ = require('lodash/fp')
+
+const setEnvVariable = require('./set-env-var')
+
+const requiredParams = ['db-password', 'hostname']
+
+if (!_.isEqual(_.intersection(_.keys(argv), requiredParams), requiredParams)) {
+  console.error('Usage: node tools/build-prod-env.js --db-password <DB_PASSWORD> --hostname <IP>')
+  process.exit(2)
+}
+
+const ENV_SAMPLE = fs.readFileSync(path.resolve(__dirname, '../.sample.env'), 'utf-8').split(os.EOL)
+fs.writeFileSync(path.resolve(__dirname, '../.env'), ENV_SAMPLE.join(os.EOL))
```

Comment:
Any special reason to read&write instead of using `fs.copyFile()` or `fs.copyFileSync()`?